BuildResidentialHPXML/measure.rb

```diff
@@ -6514,6 +6528,25 @@ def self.renumber_hpxml_ids(hpxml_bldg)
       end
     end
   end
+
+  def self.unique_hpxml_ids(hpxml_doc)
```

Comment:
FYI, I ended up adding this method into the HPXML class in my last commit [here](https://github.com/NREL/OpenStudio-HPXML/pull/1456/commits/21c1d9f32389af1ac257ea49f5050f3bcc536252). So next time you pull in that other branch, you can replace this method with a call to: `hpxml.set_unique_hpxml_ids(hpxml_doc, true)`.